pipes/rules/assembly.rules

```diff
@@ -103,7 +105,8 @@ rule orient_and_impute:
                 ono_extra.extend(["--min_pct_contig_aligned", str(config["scaffold_min_pct_contig_aligned"])])
             ono_extra = " ".join(ono_extra)
             shell("{config[bin_dir]}/assembly.py order_and_orient {input[0]} {params.refGenome} {params.scaffolded_fasta} {ono_extra} --outAlternateContigs {params.alternate_fasta}")
-            shell("{config[bin_dir]}/assembly.py impute_from_reference {params.scaffolded_fasta} {params.refGenome} {output} --newName {params.renamed_prefix}{wildcards.sample} --replaceLength {params.replace_length} --minLengthFraction {params.length} --minUnambig {params.min_unambig} --index")
+            shell("{config[bin_dir]}/assembly.py gapfill_gap2seq {params.scaffolded_fasta} {input.cleaned_reads} {params.gapfilled_fasta} --memLimitGb {resources[mem]} --maskErrors --randomSeed {config[random_seed]}")
```

Comment:
I wonder if there's any value in also gapfilling the `alternate_fasta` just for fun... or does gap2seq require the contigs to be scaffolded first (ie, it only fills gaps, it doesn't extend contigs)?

Reply:
"it only fills gaps, it doesn't extend contigs" - yes.  (IVA has a mode to extend contigs, but seems to need higher coverage than ours.)

It might make sense to also gapfill after filling from reference and refining.